src/Tablet/Sensors/Sensors.js

```diff
@@ -1,6 +1,13 @@
 import React from "react";
-import { IonContent } from "@ionic/react";
-
+import {
```

Comment:
Lots of added components that don't seem to be used. It doesn't seem like Sensors are part of the scope for Tablet Dashboard.